### ISSUE TITLE
The appender examples in README.md do not work

### ISSUE THREAD

**Issue by wtianyi** [Fuco1/emacs-lgr#2]
For example,
```
(-> (lgr-get-logger "elsa.process.file")
    (lgr-reset-appenders)
    (lgr-add-appender (lgr-appender-buffer "* Elsa Process File Log *"))
    (lgr-set-threshold lgr-level-trace)
    )
```
gives an "Unbound slot" error, while
```elisp
(-> (lgr-get-logger "elsa.process.file")
    (lgr-reset-appenders)
    (lgr-add-appender (lgr-appender-buffer :buffer "* Elsa Process File Log *"))
    (lgr-set-threshold lgr-level-trace)
    )
```
works well.

I'm not informed well enough to tell whether it's due to changes made in Emacs 29.

**Comment by Fuco1** [comment]
You're right, I'll fix it.

Edit: Actually, I can't really find this example in the readme.  Where did you see it? :D

**Comment by wtianyi** [comment]
Sorry for using an example not directly corresponding to the README examples, but the pattern is similar.

From README, one example is
```
(-> lgr
    (lgr-add-appender
     (-> (lgr-appender-file "json-logs.log")
         (lgr-set-layout (lgr-layout-json)))))
```
which would also give an "Unbound slot" error because the file name should be specified with `:file`, as
```
(-> lgr
    (lgr-add-appender
     (-> (lgr-appender-file :file "json-logs.log")
         (lgr-set-layout (lgr-layout-json)))))
```

**Comment by Fuco1** [comment]
Oh yes, you're right.  Probably in some newer version this became invalid, it was already deprecated for some time but I must have forgotten to update my habits :)  I'll update the example.

**Comment by Fuco1** [comment]
fixed in 20f3a3db9a142e86a15d14597213de23d01b772c
